README.md

```diff
@@ -316,7 +316,107 @@ The pipeline server runs the data pipeline to generate datasources (typically, d
 
 ## Production Postgres server setup
 
-Coming soon.
+### Introduction
+
+You will have to set up a [PostgreSQL](https://www.postgresql.org/) database to host relational data and web application state.
+
+### Power User
+
+By default the `power_user` account on these databases has access to all databases on the server. We do not share the `power_user` credentials with the instance. It is **recommended** to not store these credentials on your local machine in an insecure fashion. The instance has its own credentials and ability to manage its own database.
+
+### Getting Started
+
+Postgres can be installed directly on the server or via docker.
+THe database will require the `power_user` to be created as a **SUPERUSER**.
+
+#### Direct Installation
+
+Follow the [instructions](https://www.postgresql.org/download/linux/ubuntu/) to install postgres on linux (ubuntu).
+Once the database is up and running, create the `power_user` by running the below script:
+
+> Provide your own, secure password for the `power_user`, **keep it safe!**
+
+```sql
+CREATE USER "power_user" WITH
+  LOGIN
+  SUPERUSER
+  CONNECTION LIMIT -1
+  PASSWORD '[YOUR_PASSWORD]';
+COMMIT;
+```
+
+#### Docker Installation
+
+Follow the [instructions](https://docs.docker.com/engine/install/ubuntu/) to install docker on linux (ubuntu).
+Once docker is up and running, start the database by running the below script:
+
+> The terminal will prompt for the `power_user` password to use, **keep it safe!**
+
+```sh
+docker run -d --name postgres \
+    --restart always \
+    -e POSTGRES_USER=power_user \
+    -e POSTGRES_PASSWORD=$(read -s;echo $REPLY) \
+    -p 1994:5432 \
+    postgres:latest
+```
+
+### Deployment Database
+
+Postgres should now be up and running with the `power_user` **SUPERUSER**, now the database instance for the deloyment needs to be created.
+
+Running the below script, replace `[YOUR_HOSTNAME]` with the hostname/IP of your postgres instance (example: *localhost*) and `[YOUR_DATABASE_NAME]` with the database name you want to use (example: *harmony*).
```

Comment:
This can also be an internal host or internal IP.
Generally this can change from setup to setup and specifying this doesn't feel like it will add the value we are expecting.